src/olympia/addons/indexers.py

```diff
@@ -77,6 +77,7 @@ def get_mapping(cls):
                     'hotness': {'type': 'double'},
                     'icon_type': {'type': 'string', 'index': 'no'},
                     'is_disabled': {'type': 'boolean'},
+                    'is_experimental': {'type': 'boolean'},
```

Comment:
We could feasibly need to search on it - it's a psuedo-status - so I guess it should be treated how `status` is.